src/zkevm_specs/evm/execution/oog_call.py

```diff
@@ -39,18 +45,22 @@ def oog_call(instruction: Instruction):
     is_to_end_tx = instruction.is_equal(instruction.next.execution_state, ExecutionState.EndTx)
     instruction.constrain_equal(FQ(instruction.curr.is_root), is_to_end_tx)
 
+    # Both CALL and CALLCODE opcodes have an extra stack pop `value` relative to
+    # DELEGATECALL and STATICCALL.
+    rw_counter_delta = 11 + is_call + is_callcode
```

Comment:
Need to sync the update in https://github.com/privacy-scaling-explorations/zkevm-specs/pull/362, also see this commit for the fix https://github.com/han0110/zkevm-specs/commit/9c4ed00cb29c604a3c6cd89e8b986091eaeef592 which also fixes other testing files.